CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+0.5.2 (June 2023):
+------------------
+- Return all the available values for a key in a container [#77](https://github.com/ome/omero_search_engine/pull/77)
+- Return the available attributes with a container 

```

Comment:
do you have the PR number?

Reply:
The two new features are included in the same PR (i.e. 77), should I write it again in the second feature?

Reply:
Please do mention the PR or combine the 2 entries